code-the-dream-react/src/App.js

```diff
@@ -1,22 +1,33 @@
 import * as React from 'react';
+import { useEffect } from 'react';
 import { TodoList } from './TodoList.js';
 import { AddTodoForm } from './AddTodoForm.js';
  
-function App() {
-  const [todoList, setTodoList] = React.useState([]);
+function useSemiPersistentState() {
```

Comment:
great work completing the assessment for lesson 1.5. If you have any other questions please let me know.

 I don't have a lot to say other than your code is clean and readable. My only advice to you is to work on your commit history, having a single commit for the entirety of your changes makes it hard for the reviewer to understand your process. Keep it up!